audit/projects/kubernetes-public/services/container/clusters.txt

```diff
@@ -1 +1 @@
-aaa	us-central1	us-central1-c;us-central1-a;us-central1-f	6	RUNNING
+aaa	us-central1	us-central1-c;us-central1-a;us-central1-f	7	RUNNING
```

Comment:
nit about dumping clusters in a format that isn't susceptible to auto-scaling changes